### PR TITLE
Adding a separate oidc version of the image

### PR DESCRIPTION
**Pull request by lukelowery**
I was making my own image with oidc based on the instructions in the repo and I was thinking it would be simpler if the main build processes just also made that image. 

I ran both images this builds and everything seems to work. 

If everyone is happy with this approach, I can do the same for the metadata repo and update the helm chart too.

### CODE REVIEW

**Comment by feng-tao** [comment]
cc @verdan

**Comment by lukelowery** [comment]
@verdan If you are ok with this idea (even if there are some changes you want to see), I will go ahead and make the helm charts and metadata image. I have some time over the holidays to work on this again.

**Comment by verdan** [comment]
@lukelowery I am fine with the approach and the implementation 👍
(sorry for the delays in the review, somehow missed that in my emails.)

File: requirements3-oidc.txt [hunk 1]
```diff
@@ -0,0 +1,4 @@
+# A wrapper of Flask with pre-configured OIDC support.
+# License: Apache License 2.0
+# Upstream url: https://github.com/verdan/flaskoidc
+flaskoidc==0.0.2
```
Comment (feng-tao) [inline requirements3-oidc.txt]: not sure if we need another requirement file or just put it under extra_require in the setup.py file(e.g https://github.com/lyft/amundsendatabuilder/pull/180).

Comment (lukelowery) [reply requirements3-oidc.txt]: I like that idea a lot. Just pushed the commit. Could you re-approve it?

Thanks for the example!

**Pull request merged**